docs/api/index-api.rst

```diff
@@ -108,6 +108,8 @@ Functions
       - Ternary if operator
     * - :func:`shift()`
       - Shift column by a given number of rows
+    * - :func:`cumsum()`
+      - Calculate cumulative sum
```

Comment:
I guess you already have this function listed in the block below, so here you can remove it.